pkg/crc/machine/bundle/metadata.go

```diff
@@ -305,6 +311,43 @@ func getBundleDownloadInfo(preset crcPreset.Preset) (*download.RemoteFile, error
 	return downloadInfo, nil
 }
 
+func getDefaultBundleVerifiedHash(preset crcPreset.Preset) (string, error) {
+	client := &http.Client{
+		Timeout:   5 * time.Second,
+		Transport: network.HTTPTransport(),
+	}
+	res, err := client.Get(constants.GetDefaultBundleSignedHashURL(preset))
+	if err != nil {
+		return "", err
+	}
+	signedHashes, err := io.ReadAll(res.Body)
+	if err != nil {
+		return "", err
+	}
+	if err := res.Body.Close(); err != nil {
+		logging.Debug(err)
+	}
+
+	verifiedHashes, err := gpg.GetVerifiedClearsignedMsgV3(constants.RedHatReleaseKey, string(signedHashes))
+	if err != nil {
+		return "", fmt.Errorf("Invalid signature: %w", err)
+	}
+
+	logging.Debugf("Verified bundle hashes:\n%s", verifiedHashes)
+
+	lines := strings.Split(verifiedHashes, "\n")
+	for _, line := range lines {
+		if strings.Contains(line, constants.GetDefaultBundle(preset)) {
+			l := strings.Split(line, "  ")
+			if len(l) != 2 {
+				return "", errors.New("unable to find hash for default bundle")
+			}
+			return l[0], nil
+		}
```

Comment:
I'd be more strict about the expect format, 
```
if strings.HasSuffix(line, constants.GetDefaultBundle(preset) {
    sha256sum := strings.TrimSuffix(line, "  "+constants.GetDefaultBundle(preset))
    return sha256sum, nil
}
```

Reply:
thanks